tests/scripts/thread-cert/Cert_5_3_06_RouterIdMask.py

```diff
@@ -197,7 +197,6 @@ def verify(self, pv):
             filter_LLANMA().\
             filter_mle_cmd(MLE_ADVERTISEMENT).\
             filter(lambda p: {1,2,1} == set(p.mle.tlv.route64.cost) and\
-                   p.sniff_timestamp - _pkt.sniff_timestamp <= 3 and\
```

Comment:
One alternative option to ensure the timestamp check is to find two successive MLE advertisements with len(router id set) = 3, other than the strict cost check here.

Thoughts?

Reply:
> delete the second trickle time check to fix issue #5909 since there may be another advertisement with router costs set {1,0,1} which will cause the sniff time difference value larger than 3 seconds

Not quite clear about the context. But shouldn't we find the one with least sniff_timestamp first?

Reply:
the successive MLE advertisements already checking in line 182-190